py/desispec/quicklook/qlconfig.py

```diff
@@ -172,32 +189,39 @@ def qaargs(self):
         
         for PA in self.palist:
             for qa in self.qalist[PA]: #- individual QA for that PA
+                
                 params=self._qaparams(qa)
-                qaopts[qa]={'camera': self.camera, 'paname': PA, 'PSFFile': '%%PSFFile', 'amps': self.amps, 'qafile': self.dump_qa()[0][0][qa],'qafig': self.dump_qa()[0][1][qa], 'FiberMap': '%%FiberMap', 'param': params, 'qlf': self.qlf}
+                qaopts[qa]={'camera': self.camera, 'paname': PA, 'PSFFile': self.psf, 'amps': self.amps, 'qafile': self.dump_qa()[0][0][qa],'qafig': self.dump_qa()[0][1][qa], 'FiberMap': self.fibermap, 'param': params, 'qlf': self.qlf}
                 
         return qaopts 
    
     def _qaparams(self,qa):
-     
+            
         params={}
-        if qa == 'Count_Pixels':
-            params[qa]= dict(
-                             CUTLO = 100,
-                             CUTHI = 500
-                             )
-        elif qa == 'CountSpectralBins':
-            params[qa]= dict(
-                             CUTLO = 100,   # low threshold for number of counts
-                             CUTMED = 250,
-                             CUTHI = 500
-                             )
-        elif qa == 'Sky_Residual':
-            params[qa]= dict(
-                             PCHI_RESID=0.05, # P(Chi^2) limit for bad skyfiber model residuals
-                             PER_RESID=95.,   # Percentile for residual distribution
-                             BIN_SZ=0.1,) # Bin size for residual/sigma histogram
+        if self.algorithms is not None:
+            for PA in self.palist:
+                if qa in self.qalist[PA]:
+                    params[qa]=self.algorithms[PA]['QA'][qa]['PARAMS']
+
         else:
-            params[qa]= None
+            if qa == 'Count_Pixels':
```

Comment:
The params for quicklook are controlled by an input configuration object, which in normal (production) usage is set by a configuration file.  As you say, such files are important for easy modification and archiving purposes, as well as just humans reading them.  We have had the params stuff partially in place since fall, but now have implemented more completely.  The configuration file example currently in this branch is called qlconfig_dark.yaml for dark time exposures.  We will be adding  configuration examples appropriate to bright time and calibration exposure pipelines soon.  While this particular branch is not complete yet, this configuration file example will be more informative than the version in prior versions.